src/core/db/schema.ts

```diff
@@ -0,0 +1,38 @@
+import { integer, sqliteTable, text } from "drizzle-orm/sqlite-core";
+import { INTENT_STATUS } from "../../types";
+
+export const projects = sqliteTable("projects", {
+  id: text("id").primaryKey(),
+  repoPath: text("repo_path").notNull().unique(),
+  repoName: text("repo_name").notNull(),
+});
+
+export const branches = sqliteTable("branches", {
+  id: text("id").primaryKey(),
+  projectId: text("project_id")
+    .notNull()
+    .references(() => projects.id),
```

Comment:
I think it's better to add a cascade delete rule to make sure that if a project is deleted, associated branches get automatically deleted as well. It would look someting like:

```ts
.references(() => projects.id, { onDelete: 'cascade' })
```

We should also discuess how to handle intents when associated branch gets deleted.